dnnlib/comet_utils.py

```diff
@@ -0,0 +1,21 @@
+try:
+    from comet_ml import Experiment
+    comet_installed = True
+except:
+    print("Comet not installed")
```

Comment:
This prints out every time that a user tries to run code and doesn't have comet_ml installed? That would be annoying, right?

Reply:
Removed the print. Now it will be printed only when --comet flag is passed but comet not installed!